service-worker.js

```diff
@@ -111,6 +121,15 @@ function fetchEventListener(event) {
                     else if (regexpSVG.test(title)) {
                         contentType = 'image/svg+xml';
                     }
+                    else if (regexpWEBM.test(title)) {
+                        contentType = 'video/webm';
+                    }
+                    else if (regexpMP4.test(title)) {
+                        contentType = 'video/mp4';
+                    }
+                    else if (regexpOGG.test(title)) {
+                        contentType = 'video/ogg';
+                    }
                 }
                 else if (nameSpace === '-') {
                     // It's a layout dependency
```

Comment:
@mossroy Because of this test, I believe lines 142-145 below won't catch subtitles that are in the A namespace. I assume you're aware of this, but have decided that subtitles in the A namespace are an anomaly that won't be repeated in future ZIMs. I just thought I'd check.